overlord/snapstate/snapstate.go

```diff
@@ -146,22 +185,44 @@ func Remove(s *state.State, snapSpec string, flags snappy.RemoveFlags) (*state.T
 	}
 
 	// trigger remove
-	unlink := s.NewTask("unlink-snap", fmt.Sprintf(i18n.G("Deactivating %q"), snapSpec))
-	unlink.Set("snap-setup", ss)
+
+	// last task but the one holding snap-setup
+	forget := s.NewTask("forget-snap", fmt.Sprintf(i18n.G("Forgetting removed revision of %q"), snapSpec))
```

Comment:
@niemeyer I think I'm ok reusing that, but I'm wary of the name of that task, for me it doesn't imply manipulating state (remove-snap-files) atm,   wipe-snap ?